kitsune/questions/static/questions/js/components/AAQApp.jsx

```diff
@@ -39,17 +40,30 @@ export default class AAQApp extends React.Component {
       validationErrors: QuestionEditStore.getValidationErrors(),
       questionState: QuestionEditStore.getState(),
       userAuth: UserAuthStore.getAll(),
-      troubleshooting: TroubleshootingDataStore.getAll(),
+      troubleshooting: TroubleshootingDataStore.getAll()
     };
   }
 
+  setStep(step) {
+    this.setState({step: step});
```

Comment:
ES6 trick: This is can be shortened to this.setState({step});. When the key name and the name of variable for the value are the same, you only have to write it once.